libs/java/server_common/src/main/resources/messages/membership-approval.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <style></style>
+</head>
+<body style="margin: 0; background: rgba(231, 232, 231, 1.0);">
+<div class="wrapper">
+    <div class="mbrapproval">
+        <div class="logo">
+            <!-- <img src="athenz-logo-white.png" class="athenzlogowhite" alt="Athenz logo"/> -->
+            <img src="cid:logo" class="athenzlogowhite" alt="Athenz logo"/>
+        </div>
+        <div class="hdr">Membership Approval Details</div>
+        <div class="bt">Membership request has been submitted with the following details:</div>
+        <hr>
+        <table id="t01">
+            <tr><td class="ch">DOMAIN</td><td class="cv">{0}</td></tr>
+            <tr><td class="ch">ROLE</td><td class="cv">{1}</td></tr>
+            <tr><td class="ch">MEMBER</td><td class="cv">{2}</td></tr>
+            <tr><td class="ch">REASON</td><td class="cv">{3}</td></tr>
+            <tr><td class="ch">REQUESTOR</td><td class="cv">{4}</td></tr>
+        </table>
+        <hr>
+        <div class="bt">Please visit the <a href="{5}">Athenz Workflow Sceen</a> to act on the request.</div>
```

Comment:
Screen

Reply:
Will fix the typo. 
For now i did the changes only for membership notification. I can use same template for expiry notifications as well.